README.md

```diff
@@ -136,38 +144,184 @@ if let success = response as? AuthenticatePasskeyResult.Success {
 
 The creation payload for creating a passkey is a String obtained by requesting your backend a challenge for registering a user, it uses the JSON schema:
 ```
-{"rp":{"id":"your_backend","name":"PasskeySample"},"user":{"id":"WUV...5Ng","name":"1234567890","displayName":"1234567890"},"challenge":"WUY...jZQ","pubKeyCredParams":[{"type":"public-key","alg":-7}],"timeout":600000,"excludeCredentials":[],"authenticatorSelection":{"authenticatorAttachment":"platform","requireResidentKey":false,"residentKey":"preferred","userVerification":"preferred"},"attestation":"none"}
+{
+  "rp": {
+    "id": "your_backend",
+    "name": "PasskeySample"
+  },
+  "user": {
+    "id": "WUV...5Ng",
+    "name": "1234567890",
+    "displayName": "1234567890"
+  },
+  "challenge": "WUY...jZQ",
+  "pubKeyCredParams": [
+    {
+      "type": "public-key",
+      "alg": -7
+    }
+  ],
+  "timeout": 600000,
+  "excludeCredentials": [],
+  "authenticatorSelection": {
+    "authenticatorAttachment": "platform",
+    "residentKey": "preferred",
+    "userVerification": "preferred"
+  }
+}
 ```
 
+1. **rp** (Relying Party - Server Details)
```

Comment:
is it worth to add a link (if exist) to find more documentation about this payload json and its value?

Reply:
Added link to WebAuthn in documentation